src/printer-markdown.js

```diff
@@ -93,7 +93,7 @@ function genericPrint(path, options, print) {
         "```",
         node.lang || "",
         hardline,
-        node.value,
+        getFormattedCode(path, options),
```

Comment:
No way to use multiparser here?

Reply:
Actually, I don't know how to use multiparser, need some helps. 😅

Reply:
* Add another case to `getSubtreeParser()` for `markdown`
* call into a new function `fromMarkdown(path)`
* Switch over the `node.type` like the other functions
* For `case "code"`, do something like this:

    ```js
    const parser = getParserName(node.lang);
    if (parser) {
      return {
        options: { parse },
        transformDoc: doc => concat([
          "```",
          node.lang,
          hardline,
          doc,
          "```"
        ]),
        text: node.value
      }
    }
    ```

Reply:
It should handle indentation automatically as it ends up being one big `doc`.

Reply:
@azz thanks, it works! ❤️